src/env/index.ts

```diff
@@ -22,3 +22,5 @@ export const getEnv = () => env || getBrowserEnv();
 export const getDocument = (): Document => getEnv().document;
 
 export const getWindow = (): Window | DOMWindow => getEnv().window;
+
+export const getDOMParserFactory = () => getEnv().DOMParser;
```

Comment:
I prefer that this method resides in the env object so that the window object is not accessed at import time.
We don't have proper tests for it but I can image that in SSR it will be undefined